### DISCUSSION TITLE
Stable Diffusion Colab Notebook losting the connection all the time (it's general?)

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1627] [Q&A]
Hey guys,

Everytime I need to use the stop button and start the Stable-Diffusion again because it keeps disconnecting, this is happening with you guys too?

![image](https://user-images.githubusercontent.com/121369033/221282295-59c25db0-eb0e-44d3-ad97-38b345d59ae7.png)

Another thing is 70% of time that I generate multiple image don't show the preview, just generates and shows later.

**Comment by cutecoelho** [comment]
Same thing for me, it's annoying sometimes. Hope it could be fix

**Comment by animeking527** [comment]
I don't know if this is what solved it for me or if it was just a coincidence. But after cleaning up my sd folder a bit I noticed it happened less. I removed textual inversion embeddings & loras that weren't compatible and turned off extensions other than controlnet & additional networks. Generation seemed to not hang for as long before starting and the time out connection was occurring less.
